config/samples/test_v1beta1_horizontest.yaml

```
@@ -11,6 +11,7 @@ metadata:
 spec:
   containerImage: ""
   # debug: false
+  storageClass: "lvms-local-storage"
```

We should put here the default value which is `local-storage`. 
```suggestion
  storageClass: "local-storage"
```